rust/numaflow-core/src/config.rs

```diff
@@ -192,20 +245,77 @@ impl Settings {
                 .and_then(|metadata| metadata.name)
                 .ok_or_else(|| Error::ConfigError("Mono vertex name not found".to_string()))?;
 
-            settings.sdk_config.is_transformer_enabled = mono_vertex_obj
+            settings.transformer_config = match mono_vertex_obj
                 .spec
                 .source
+                .as_deref()
                 .ok_or(Error::ConfigError("Source not found".to_string()))?
                 .transformer
-                .is_some();
+            {
+                Some(_) => Some(TransformerConfig::default()),
+                _ => None,
+            };
 
-            settings.sdk_config.is_fallback_enabled = mono_vertex_obj
+            settings.udsource_config = match mono_vertex_obj
```

Comment:
This `settings` needs to be refactored later on for Vertex.

Reply:
yes, config needs a bit of rework.